### PR TITLE
Fixed issue #3681

### PR DESCRIPTION
**Pull request by Anjali-NEC**
Fix issue #3681

### CODE REVIEW

File: src/app/contextBroker/version.h [hunk 1]
```diff
@@ -28,6 +28,6 @@
 
 
 
-#define ORION_VERSION "2.4.0-next"
+#define ORION_VERSION "2.4.0_20200802203650"
```
Comment (fgalan) [inline src/app/contextBroker/version.h:31]:
```suggestion
#define ORION_VERSION "2.4.0-next"
```

File: test/functionalTest/cases/0000_large_requests/notification_different_sizes.test [hunk 2]
```diff
@@ -240,9 +240,9 @@ Date: REGEX(.*)
 ** Too large errors:
 1
 ** Notification sizes:
-Sending message 1 to HTTP server: sending message of REGEX((19485|19490|19486|19491)) bytes to HTTP server
-Sending message 2 to HTTP server: sending message of REGEX((21485|21490|21486|21491)) bytes to HTTP server
-Sending message 3 to HTTP server: sending message of REGEX((8100487|8100492|8100488|8100493)) bytes to HTTP server
+Sending message 1 to HTTP server: sending message of 19500 bytes to HTTP server
+Sending message 2 to HTTP server: sending message of 21500 bytes to HTTP server
+Sending message 3 to HTTP server: sending message of 8100502 bytes to HTTP server
```
Comment (fgalan) [inline test/functionalTest/cases/0000_large_requests/notification_different_sizes.test:245]: Please don't collapse all the 4 cases into just one, taking into account the note bellow:

```
# NOTE: the different sizes above is to cover for different sizes in the version string
#
#       d.d.d         (+0)
#       d.d.d-next    (+5)
#       d.dd.d        (+1)
#       d.dd.d-next   (+6)
```

File: src/app/contextBroker/version.h (hunk 1 above)
Comment (Anjali-NEC) [reply src/app/contextBroker/version.h]: Updated

**Review by fgalan — APPROVED** [review]
LGTM. Thanks you very much for your contribution!

There are some small things to fix, but I'll take care of them after merging this PR.

**Pull request merged**